docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -0,0 +1,72 @@
+[role="xpack"]
+[[ml-dfa-phases]]
+=== How a {dfanalytics-job} works
+
+This page provides high-level background information on how {dfanalytics-jobs} 
+work, what kind of phases the jobs have, and what happens during the individual 
+phases.
+
+
+[discrete]
+==== The phases of a {dfanalytics-job}
+
+During its life cycle, a {dfanalytics-job} goes through four phases regardless 
+of the {dfanalytics} type it performs. The phases are discrete steps of the 
+{dfanalytics-job} that is essentially a persistent {es} task. The four phases 
+are the following:
+
+* reindexing,
+* loading data,
+* analyzing,
+* writing results.
+
+Let's take a look at the phases one-by-one.
+
+
+[discrete]
+===== Reindexing
+
+During the reindexing phase the documents from the source index or indices are 
+copied to the destination index. If the destination index does not exist, then 
```

Comment:
Perhaps this is simpler?
```suggestion
copied to the destination index. If you want to define settings or mappings, you can create the index yourself. Otherwise,
```